mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
@rljacob, sorry - apparently I cleaned-up the white space in m_SPMDutils.F90 since I did my last commit (to synchronize with what is in CIME and ACME?) and this got added in this latest commit. You probably want this, but not necessarily as part of this PR? Please tell me what you would like me to do, if anything.